README.md

```diff
@@ -1062,6 +1062,37 @@ providers:
 
 ```
 
+## KeyPass
+
+### Authentication
+
+requires the following environment variables to be set:
+
+`KEYPASS_PASSWORD`: Password database credentials
+`KEYPASS_DB_PATH`: Database path
+
+### Features
+
+* Sync - `no`
+* Mapping - `yes`
+* Modes - `read`
+
+### Example Config
+
+```yaml
+
+providers:
+  keypass:
+    env_sync:
+    path: redis/config
+    # source: Optional, by default, return password field. To get all entry fields set 'all-fields
```

Comment:
I'm wondering about `all-fields`, given that it's a magic constant, perhaps by default return all fields and specify `password`, inverting the logic?

If not maybe just to use `all` as the key.

Reply:
Agree @jondot, i have changed the default to be all fields.